build_files/base/05-override-install.sh

```diff
@@ -15,6 +15,13 @@ install -c -m 0755 /tmp/starship /usr/bin
 # shellcheck disable=SC2016
 echo 'eval "$(starship init bash)"' >> /etc/bashrc
 
+# Nerdfont symbols
+# to fix motd and prompt atleast temporarily
+ghcurl "https://github.com/ryanoasis/nerd-fonts/releases/latest/download/NerdFontsSymbolsOnly.zip" --retry 3 -o /tmp/nerdfontsymbols.zip
+unzip /tmp/nerdfontsymbols.zip -d /tmp
+mkdir -p /usr/share/fonts/nerd-fonts/NerdFontsSymbolsOnly/
+mv /tmp/*.ttf /usr/share/fonts/nerd-fonts/NerdFontsSymbolsOnly/
```

Comment:
```suggestion
mv /tmp/SymbolsNerdFont*.ttf /usr/share/fonts/nerd-fonts/NerdFontsSymbolsOnly/
```
Let's narrow this glob, just in case `/tmp` happens to have random fonts in it (unlikely, but technically possible).


<img width="1393" height="989" alt="image" src="https://github.com/user-attachments/assets/94b5991c-7964-4b3d-966a-c03cac04173f" />

Reply:
Right, didnt even think about that